### DISCUSSION TITLE
Make show/hide palette/sidebar an always visible button rather than a slide-out button

### DISCUSSION THREAD

**Discussion by jpravetz** [node-red/designs#84] [Feature Requests]
This is a _nice to have_ UI feature request.

Currently, to click the toggle the palette or sidebar visibility buttons, you must mouse to the hover spot under which the button resides, wait for the button to slide out, then click it. This is made somewhat less convenient by the fact that the hover spot is not always identified (on the right there is a vertical bar at middle height, on the left there doesn't seem to be).

It would be more convenient, and would save me 10s of seconds every day, if those button were exposed without requiring a hover. An example implementation that is better is the one used by Adobe Acrobat.

Current Node-RED implementation (must hover, wait, then click):
 
![Screenshot 2023-09-13 at 1 41 21 PM](https://github.com/node-red/designs/assets/154432/7cbb0712-3f51-472a-bf63-e9277eacbea4)

Adobe Acrobat implementation (button is always visible):

<img width="69" alt="Screenshot 2023-09-13 at 1 45 54 PM" src="https://github.com/node-red/designs/assets/154432/6ada32e1-1ca7-4c76-a83d-567f91b03125">

Home Assistant also has an implementation which works well when collapsing their left-hand sidebar:

![Screenshot 2023-09-13 at 1 50 28 PM](https://github.com/node-red/designs/assets/154432/6294f329-5d88-497d-b981-e031ecc64e33)

![Screenshot 2023-09-13 at 1 50 21 PM](https://github.com/node-red/designs/assets/154432/eac31223-2c5b-4a98-8488-7a41272f3041)

**Comment by knolleary** [comment]
An early iteration did put a button in the thin edge of the separator. But we found it was a bit too small to be useful - and wasn't really discoverable as it was hard to spot. That said, I can see the benefit of having an always visible toggle for it - it's just a matter of figuring out where it goes without encroaching on what's there already.

**Comment by jpravetz** [comment]
Thanks for the thoughtful consideration. On the point of discoverability, it did take me quite awhile before I discovered I could collapse the sidebars, due to the absence of a visible button with which to do so. I think I first found it via the right-hand vertical bar, looking to use that to resize that panel.
